_sass/_footer.scss

```diff
@@ -26,6 +26,15 @@
   border-top: 1px solid #EEE;
   text-align: center;
   box-sizing: border-box;
-  padding: 10px;
+  padding: 10px 10px 30px 10px;
   margin-top: 30px;
+  position: relative;
+  a.next {
```

Comment:
selectors with specified classes should not start with their base elements. This should just be `.next` :)